core/src/utils/animation/animation-utils.ts

```diff
@@ -24,12 +24,25 @@ const convertHyphenToCamelCase = (str: string) => {
   return str.replace(/-([a-z])/ig, g => g[1].toUpperCase());
 };
 
+let animationPrefix: string | null = null;
```

Comment:
Ok sorry last thing -- we typically prefer undefined over null for consistency
```suggestion
let animationPrefix?: string;
```

Reply:
Just wanted to clarify: there's no bug here, but we usually prefer using `undefined` over `null`where applicable so it would be good to remain consistent.

Reply:
My typescript (3.7.2) did not like a global variable to be marked as optional with `?`. 
Error:(27, 5) TS7034: Variable 'animationPrefix' implicitly has type 'any' in some locations where its type cannot be determined.
Error:(27, 20) TS1005: ',' expected.
Error:(27, 21) TS1134: Variable declaration expected.

Reply:
Oh strange... ok `string | undefined` is fine. thanks!